app/jobs/bulkrax/import_work_job.rb

```diff
@@ -4,20 +4,25 @@ class ImportWorkJob < ApplicationJob
 
     def perform(*args)
       entry = Entry.find(args[0])
+      importer_run = ImporterRun.find(args[1])
       begin
-        entry.build
+        reschedule(entry.id, importer_run.id) if entry.build.blank?
         entry.save
       rescue StandardError => e
-        ImporterRun.find(args[1]).increment!(:failed_records)
+        importer_run.increment!(:failed_records)
         raise e
       else
         if entry.last_exception
-          ImporterRun.find(args[1]).increment!(:failed_records)
+          importer_run.increment!(:failed_records)
           raise entry.last_exception
         else
-          ImporterRun.find(args[1]).increment!(:processed_records)
+          importer_run.increment!(:processed_records)
         end
       end
     end
+
+    def reschedule(entry_id, run_id)
+      ImportWorkJob.perform_later(entry_id, run_id)
```

Comment:
probably should put a delay to not flood the workers with stuff they can't actually do anything with